package.json

```
@@ -9,6 +9,14 @@
     "lint": "next lint"
   },
   "dependencies": {
+
+    "@tiptap/extension-link": "^2.22.3",
+    "@tiptap/extension-mention": "^2.22.3",
+    "@tiptap/extension-placeholder": "^2.22.3",
+    "@tiptap/react": "^2.22.3",
+    "@tiptap/starter-kit": "^2.22.3",
+    "class-variance-authority": "^0.7.1",
```

Biome (1.9.4)

[error] The key class-variance-authority was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)